src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -0,0 +1,223 @@
+
+- `"none"` (the default): No link capturing; links clicked leading to this PWA scope navigate as
+  normal without opening a PWA window.
+- `"new-client"`: Each link clicked leading to this PWA scope opens a new PWA window at that URL.
+- `"existing-client-navigate"`: When a link is clicked leading to this PWA scope, the user agent
+  finds an existing PWA window, and causes it to navigate to the opened URL. If more than one PWA
+  window exists, the browser may choose one arbitrarily. This behaves like `"new-client"` if no
+  window is currently open. 🚨 Careful! This option potentially leads to data loss, as pages can be
+  arbitrarily navigated away from. Sites should be aware that they are opting into such behavior by
+  choosing this option. This option works best for "read-only" sites that do not hold user data in
+  memory, such as music players. If the page being navigated away from has a
+  [`beforeunload` event](https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload),
+  the user would see the prompt before the navigation completes.
+- `"existing-client-event"`: When a link is clicked leading to this PWA scope the user agent finds
+  an existing PWA window and fires a `launch` event in that window's top-level context, containing
+  the launched URL. If more than one PWA window exists, the browser may choose one arbitrarily. This
+  behaves as `"new-client"` if no window is currently open. This is intended for more advanced
+  sites, which inspect the launched URL and use it to load data into the current browsing context,
+  without a navigation. For example, a site that allows multiple documents to be loaded in the same
+  window can pop open a new sub-window in response to the navigation.
+
+{% Aside 'objective' %} There is ongoing discussion about adding a `"serviceworker"` option in the
+future that does not open a window at all, but instead fires a `launch` event in the service
+worker's context. See the
+[`launch` event explainer](https://github.com/WICG/sw-launch/blob/main/explainer.md) for details.
+{% endAside%}
+
+## Demo
+
+The demo for Declarative Link Capturing actually consists of two demos that interact together:
+
+1. [https://continuous-harvest-tomato.glitch.me/](https://hill-glitter-tree.glitch.me/)
+1. [https://hill-glitter-tree.glitch.me/](https://hill-glitter-tree.glitch.me/)
+
+The screencast below shows how the two interact together. They show two different behaviors,
+`"new_client"` and `"existing_client_navigate"`. Be sure to test the apps in different states,
+running in a tab or as installed PWA, to see the difference in behavior.
+
+{% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/pj3ehpntEg50WcnA2khM.webm", autoplay=true, muted=true, playsinline=true, loop=true %}
+
+## Security and permissions
+
+The Chromium team has designed and implemented Declarative Link Capturing using the core principles
```

Comment:
I don't really know what this paragraph is saying. It doesn't have much to do with security / permissions. I would just take this out.

Reply:
This is boilerplate that we use in certain API articles to make people aware of our efforts around privacy and security. Have you looked at [the linked content](https://chromium.googlesource.com/chromium/src/+/lkgr/docs/security/permissions-for-powerful-web-platform-features.md)?

Reply:
It's boilerplate, but something we consistently tend to emphasize to point people at the fact that we thought about this more deeply and that it doesn't just come out of thin air.

Reply:
Yes. What he said. 'Boilerplate' was a bad word choice. It implies overuse.